src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -839,6 +840,7 @@ public function __construct()
 
         $code .= "\n        \$this->services = array();\n";
         $code .= $this->addMethodMap();
```

Comment:
I think we forgot `$code .= $this->addPrivateServices();` here (see `addConstructor`), should we add it?